src/main/java/com/epam/reportportal/auth/util/PersonalProjectService.java

```diff
@@ -68,7 +68,7 @@ public PersonalProjectService(ProjectRepository projectRepository,
    * @return Corresponding personal project name
    */
   @VisibleForTesting
-  private String generatePersonalProjectName(String username) {
+  String generatePersonalProjectName(String username) {
```

Comment:
no need keeping `VisibleForTesting` annotation then

Reply:
VisibleForTesting make method private in compile level, still package for testing